io.embrace.sdk/Editor/EditorUtilities/Validator.cs

```diff
@@ -38,10 +38,13 @@ public static void ValidateConfiguration(EmbraceConfiguration config)
             {
                 throw new UnityEditor.Build.BuildFailedException("EmbraceSDK: No config scriptable object found, add the EmbraceSDK component via Component/Scripts and configure it with your API Token via the Unity Inspector");
             }
+            #if !UNITY_IOS
+            // iOS does not require an App ID.
```

Comment:
Android doesn't either if exporters are configured, but I'm assuming that's getting addressed in future work?

Reply:
The plan is to address that in future work at this time, as the Unity API doesn't currently allow for exporter configuration in this version.